test/parallel/test-net-server-connections-child-null.js

```diff
@@ -31,10 +36,14 @@ if (process.argv[2] === 'child') {
 
     server.listen(0, common.mustCall(() => {
       const connect = net.connect(server.address().port);
-
```

Comment:
Unrelated white-space change?

Reply:
@Trott, I agree, would you like me to add the space?

Reply:
just remove this change, we wouldn't like to break `git blame`

Reply:
This change is ready, thanks for the feedback @Himself65